processor.py

```diff
@@ -6,13 +6,24 @@ def __init__(self, number):
     def getid(self):
         return self.number[2:]
 
-    # Values must be passed as a string representation of a hex
-    # value, or as a hex value. They should not be passed as ints.
     def setvalue(self, value):
+        """Values must be passed as a string representation of a hex value
+        or as a hex value. They should not be passed as ints
+
+        Args:
+            value (str): a hex representation of the contents of the cell
+
+        """
         self.value = int(value, 16)
 
-    # Values will be returned as a base 16 int.
+
     def getvalue(self):
+        """Values will be returned as base 16 int.
```

Comment:
This should be a description, not just a technicality on how to use the function.